app/src/app/store/mapEditSubs.ts

```diff
@@ -6,6 +6,8 @@ import {shallowCompareArray} from '../utils/helpers';
 import {updateAssignments} from '../utils/api/queries';
 import {queryClient} from '../utils/api/queryClient';
 
+export let allowSendZoneUpdates = true
```

Comment:
This seems very un-React-y but as I understand it keeps additional updates from occurring during shatter/heal events. If it's just for shattering, it might be helpful to put a comment here

Reply:
PP: Agreed, we should add a comment here. Something like:

```javascript
// We use basic plain ol javascript variables here to ensure that any
// asynchronous operations / subscriptions on the state.
// This is set to false when operations are occurring that heal or break vtds
// during an undo/redo operation. This can create intermediary state changes to the zone updates
// so it's important that we avoid sending those to the backend.
export let allowSendZoneUpdates = true